pkg/kafka/policy.go

```diff
@@ -227,6 +265,13 @@ func (req *RequestMessage) ruleMatches(rule api.PortRuleKafka) bool {
 		return matchOffsetFetchReq(val, rule)
 	case *proto.ConsumerMetadataReq:
 		return true
+	case nil:
```

Comment:
I think we should also add `default` here as we will want to not allow an apiKey if the apiKey contains a topic, the parser understands the message but we are not listing it above.

Reply:
Sure. I am returning `false` because we should not allow traffic/request if policy is in place.

Reply:
done